apps/nerves_hub_web_core/lib/nerves_hub_web_core/products/product.ex

```diff
@@ -18,6 +19,7 @@ defmodule NervesHubWebCore.Products.Product do
     has_many(:firmwares, Firmware)
     has_many(:product_users, ProductUser)
     has_many(:users, through: [:product_users, :user])
+    has_one(:jitp, CACertificate.JITP)
```

Comment:
```suggestion
    has_one(:ca_certificate, CACertificate)
    has_one(:jitp, CACertificate.JITP, through: :ca_certificate)
```

Is this needed? If so, maybe we are better server using `through` or preloading?

Reply:
This isn't actually used anywhere, i just wanted to be consistent with the relationships in case we need it in the future.